src/parser/util.js

```diff
@@ -86,3 +87,25 @@ pp.unexpected = function (pos, messageOrType = "Unexpected token") {
   }
   this.raise(pos != null ? pos : this.state.start, messageOrType);
 };
+
+pp.expectPlugin = function (name) {
+  if (!this.hasPlugin(name)) {
+    const pluginInfo = plugins[name];
+    const pointers = [];
+    if (pluginInfo) {
+      if (pluginInfo.babelSyntaxPlugin) {
+        pointers.push(`babel-plugin-${pluginInfo.babelSyntaxPlugin}`);
+      }
+      if (pluginInfo.babelTransformPlugins) {
+        pointers.push(...pluginInfo.babelTransformPlugins.map(
+          (shortName) => `babel-plugin-${shortName}`
+        ));
+      }
+    }
+    const leadingPointersOr = pointers.length ? ` ${pointers.join(", ")} or ` : " ";
```

Comment:
How would people feel about skipping this `pointers` logic and having `babylon` just throw an error with the plugin name in the message, and a `.missingPlugin` property?

They we can have the try/catch in `babel-core` catch the exception and handle mapping the error to a Babel plugin? That way we don't need Babylon to be aware of Babel stuff.